Backend/kobrastocks/services.py

```diff
@@ -82,21 +83,28 @@ def retrieve_data(ticker):
 
 def add_indicators(dataframe, MACD=False, RSI=False, SMA=False, EMA=False, ATR=False, BBands=False, VWAP=False):
     indicators = []
-
+    indicator_names=[]
```

Comment:
We shouldn't need the indicator names array because we have the indicators array

Reply:
It is to set up the subsets for dropna